packages/core/src/Utils/Analytics/index.ts

```diff
@@ -17,45 +17,53 @@ export const AnalyticsInitializer = async () => {
             .then(res => res.json())
             .catch(() => FIREBASE_INIT_DATA);
         if (process.env.RUDDERSTACK_KEY && flags?.tracking_rudderstack) {
+            const utm_cookie = Cookies.get('utm_data');
             const ppc_campaign_cookies =
-                Cookies.getJSON('utm_data') === 'null'
+                utm_cookie === 'null' || !utm_cookie
                     ? {
                           utm_source: 'no source',
                           utm_medium: 'no medium',
                           utm_campaign: 'no campaign',
                           utm_content: 'no content',
                       }
-                    : Cookies.getJSON('utm_data');
+                    : JSON.parse(utm_cookie);
 
-            const client_information = Cookies.getJSON('client_information');
+            const client_information = Cookies.get('client_information')
+                ? JSON.parse(Cookies.get('client_information') || '')
```

Comment:
we are checking if cookie exists in line 31. So do we need `|| ''` in line 32 again?